packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{! template-lint-disable no-invalid-role }}
+{{! template-lint-disable require-context-role }}
+
+<div class={{this.classNames}} ...attributes>
+  <div class="hds-tabs__tablist-wrapper">
```

Comment:
I appreciate this may be of use in the future (for things like tab scrolling, if we decide to implement such a feature), but with the changes introduced in this PR I don't see the need for it. Am I missing something?

Reply:
I had the same doubt but then I realized that this wrapper is necessary once the indicator at the bottom of the tabs is introduced. Am I right @KristinLBradley ?

Reply:
I'm including it because I think it will probably be needed to wrap the current tab indicator and the overflow navigation.

Reply:
If we end up releasing without overflow navigation and it turns out I don't need it for the indicator, etc. then I'll refactor to remove it before final release...